### PR TITLE
Todi defaults

### PR DESCRIPTION
**Pull request by StongeEtienne**
Simplified TODI script, added details and default values in the help
 for issue #408

### CODE REVIEW

File: scripts/scil_compute_todi.py [hunk 1]
```diff
@@ -33,39 +35,39 @@ def _build_arg_parser():
     p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
                                 formatter_class=argparse.RawTextHelpFormatter)
 
-    p.add_argument('in_bundle',
+    p.add_argument('input_tracto',
```
Comment (GuillaumeTh) [inline scripts/scil_compute_todi.py:38]: Do you have any use case where it is not a bundle ? Moreover, the standard in scilpy is in_. Can you change it ?

File: scripts/scil_compute_todi.py [hunk 2]
```diff
@@ -33,39 +35,39 @@ def _build_arg_parser():
...
-    p.add_argument('--out_lw_tdi',
-                   help='Output length-weighted TDI map.')
+    p.add_argument('--out_tdi',
+                   help='Output Track Density Image (TDI).')
 
-    p.add_argument('--out_lw_todi',
-                   help='Output length-weighted TODI map.')
+    p.add_argument('--out_todi_sf',
+                   help='Output TODI, with SF (each directions\n'
+                        'on the sphere), require a lot of memory)')
```
Comment (GuillaumeTh) [inline scripts/scil_compute_todi.py:58]: The parenthesis seems to not fit.

File: scripts/scil_compute_todi.py [hunk 3]
```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 """
-Compute a length-weighted Track Orientation Density Image (TODI).
-This script can afterwards output a length-weighted Track Density Image
-(TDI) or a length-weighted TODI, based on streamlines' segments.\n\n
+Compute a Track Orientation Density Image (TODI).
+Each segment of the streamlines is weighted by length,
```
Comment (GuillaumeTh) [inline scripts/scil_compute_todi.py:5]: Sound weird. Maybe something like this: Each segment of the streamlines is weighted by length (to support compressed streamlines)

**Review by GuillaumeTh — CHANGES_REQUESTED** [review]

File: scripts/scil_compute_todi.py (hunk 3 above)
Comment (mdesco) [reply scripts/scil_compute_todi.py:5]: Agree with this suggestion

File: scripts/scil_compute_todi.py (hunk 2 above)
Comment (mdesco) [reply scripts/scil_compute_todi.py:58]: remove ) after sphere

require -> requires

File: scripts/scil_compute_todi.py [hunk 4]
```diff
@@ -33,39 +35,39 @@ def _build_arg_parser():
...
 
-    p.add_argument('--sh_normed', action='store_true',
-                   help='Normalize SH.')
+    p.add_argument('--normalize_per_voxel', action='store_true',
+                   help='Normalize each SF or SH at each voxel.')
 
-    p.add_argument('--smooth', action='store_true',
-                   help='Smooth todi (angular and spatial).')
+    p.add_argument('--smooth_todi', action='store_true',
+                   help='Smooth TODI (angular and spatial).')
```
Comment (mdesco) [inline scripts/scil_compute_todi.py:70]: please add the defaults for the boolean options too. [false]

**Review by mdesco — COMMENTED** [review]
Minor edits from Guillaume and I, then good to go.

File: scripts/scil_compute_todi.py (hunk 1 above)
Comment (StongeEtienne) [reply scripts/scil_compute_todi.py]: It can be used for any tractogram, similar to a TDI map or image

File: scripts/scil_compute_todi.py (hunk 2 above)
Comment (StongeEtienne) [reply scripts/scil_compute_todi.py]: done

File: scripts/scil_compute_todi.py (hunk 3 above)
Comment (StongeEtienne) [reply scripts/scil_compute_todi.py]: done

**Comment by mdesco** [comment]
@StongeEtienne  Can you please add the boolean defaults to false and then I approve. Otherwise, we cannot know they are boolean options

File: scripts/scil_compute_todi.py (hunk 4 above)
Comment (StongeEtienne) [reply scripts/scil_compute_todi.py:70]: @arnaudbore is it common to add default to "action='store_true',"

Comment (mdesco) [reply scripts/scil_compute_todi.py:70]: I want the user to know it is a boolean option. Right now impossible to know with --smooth_todi needs to be called with a float or a true or false. Just find a way for the user to call it with 
--smooth_todi true 
and not --smooth_todi 0.25 or whatever

Comment (StongeEtienne) [reply scripts/scil_compute_todi.py]: done

**Review by mdesco — APPROVED** [review]

**Review by GuillaumeTh — APPROVED** [review]

**Pull request merged**